### PR TITLE
Investment search

### PR DESCRIPTION
**Pull request by teneightfive**
The main piece of work here is to add investment projects to the global search. It includes a refactor to search to make it easier to use in other sub-apps.

## What it looks like

image

### CODE REVIEW

File: src/templates/_layouts/datahub-base.njk [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 {% from "_macros/form.njk" import EntitySearchForm, Form with context %}
 {% from "_macros/form.njk" import TextField, MultipleChoiceField %}
+{% from "_macros/address.njk" import addressFormatter %}
```
Comment (tyom) [inline src/templates/_layouts/datahub-base.njk]: Should this be a macro or a filter? Perhaps a separate PR.

File: src/templates/_layouts/search.njk [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{% endblock %}
+
+
+{% block main_grid_left_column %}
+  &nbsp; {# // TODO filters #}
+{% endblock %}
+
+
+{% block main_grid_right_column %}
+  {{ super() }}
```
Comment (tyom) [inline src/templates/_layouts/search.njk]: Is this needed? If so should it be before summary?

File: src/templates/_layouts/search.njk [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  {% if results %}
```
Comment (tyom) [inline src/templates/_layouts/search.njk]: Should the block be called `search_results` to be more explicit about its context? Same for the others.

File: src/templates/_layouts/search.njk [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    {% block results_summary %}
+      {% component 'results-summary', {
+        resultType: searchEntity | replace('_', ' '),
+        total: results.count,
+        searchTerm: searchTerm
+      } %}
+    {% endblock %}
+
+    {% block results %}
```
Comment (tyom) [inline src/templates/_layouts/search.njk]: Should the block be called `search_results` to be more explicit about its context? Same for the others.

File: src/templates/_layouts/datahub-base.njk (hunk 1 above)
Comment (teneightfive) [inline src/templates/_layouts/datahub-base.njk]: Could potentially be moved to a filter. Definitely a separate PR though.

Comment (teneightfive) [inline src/templates/_layouts/datahub-base.njk]: There is already an issue linked to this which I've added a comment to: #169

**Pull request merged**